IdentitySdkCore/IdentitySdkCore/Classes/models/requests/LoginRequest.swift

```diff
@@ -1,14 +1,16 @@
 import Foundation
 
 public class LoginRequest: Codable, DictionaryEncodable {
-    public let username: String
+    public let email: String?
+    public let phoneNumber: String?
```

Comment:
I don't really like this kind of type modeling. It's also prevalent in `ciam-app`. Basically here we can have either an `email` or a `phoneNumber`, but the type still allows representing a case where both are absent.